index.rst

```diff
@@ -1,59 +1,367 @@
+   | <instrument>/calib/<ticket>/*                     | unspecified | Calibration production runs.                                                  |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | [<instrument>/]runs/<target>/<release>/<ticket>   | CHAINED     | Public outputs of processing data <target> with <release> on <ticket>.        |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | [<instrument>/]runs/<target>/<release>/<ticket>/* | unspecified | Private intermediates of processing data <target> with <release> on <ticket>. |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | refcats                                           | CHAINED     | All reference catalogs (distinguished by dataset type).                       |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | refcats/<ticket>                                  | RUN         | One reference catalog, ingested and sharded on <ticket>.                      |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | skymaps                                           | RUN         | All skymap definition datasets (distinguished by data ID).                    |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+   | u/<user>/*                                        | unspecified | Experimental/development processing by <user>.                                |
+   +---------------------------------------------------+-------------+-------------------------------------------------------------------------------+
+
+.. _daf_butler documentation: https://pipelines.lsst.io/v/weekly/modules/lsst.daf.butler/organizing.html
+
+.. _collections-per-instrument:
+
+Per-instrument collections
+--------------------------
+
+Raw and calibration data associated with a particular instrument is organized into collections that start with the "short" instrument name, e.g. "HSC" or "LSSTCam-imSim", followed by a slash.
+The naming conventions for these collections are codified by the `lsst.obs.base.Instrument`_ class's ``make*Name`` methods.
+The highest-level collections are always defined as ``CHAINED`` collection "pointers" to other versioned collections.
+
+In the case of raw data, we propose three levels of collections:
+
+ - ``<instrument>/raw/all``: the ``RUN`` collection into which all raws for that instrument are originally ingested.
```

Comment:
Should we allow users to ingest raws into their `u/<user>` collections, and if so should we specify the convention for naming userspace raw collections; possibly `/u/<user>/<instrument>/raw`? If we recommend against that (can we even prevent it?), what is the process for getting new data ingested if you are not in the `<instrument>_raw` role group?

Reply:
I do strongly recommend against that, but it's hard to prevent right now; if I can away to make it impossible without assuming a non-user role, I will.  The big problem is that raw ingestion does not just ingest datasets - it also inserts `exposure` dimension rows, and if some user does that _before_ it is done officially, and something goes wrong, it'll be hard to clean things up in order to let the official ingestion run proceed.

> what is the process for getting new data ingested if you are not in the <instrument>_raw role group?

I think this'll be a bit different for each instrument, but to first order, I think it's basically, "ask your friendly neighborhood member of that group".

Reply:
@mrawls has strong opinions here, regarding science users who want to process e.g. last night's DECam data right away, when we don't have an official ingest for it. That may be something that we explicitly do not support on this system as it is (and as you say, we may want to make it impossible), but it is something we'll want to think about for the RSP.